vtysh/vtysh_ovsdb_router_context.c

```diff
@@ -195,13 +195,23 @@ static void vtysh_router_context_bgp_print_nbr_cfg(vtysh_ovsdb_cbmsg_ptr p_msg,
 
     if (nbr_table->update_source) {
         if (!pgroup_ptr ||
-            strcmp(nbr_table->update_source, pgroup_ptr->update_source)  != 0){
+            strcmp(nbr_table->update_source, pgroup_ptr->update_source) != 0){
             vtysh_ovsdb_cli_print(p_msg,"%4s %s %s %s %s", "", "neighbor",
                                  neighbor,
                                  "update-source",
                                  (nbr_table->update_source));
         }
     }
+
+    if (nbr_table->local_as) {
```

Comment:
Do you suggest to do it for all the cases in this function or just local-as?

To keep the common style we the proper way is to replace all such checks in this file(or at least in this function) or let it be as it is now. I see no major point to apply change that you've suggested.